docs/about/index.html

```diff
@@ -52,9 +56,15 @@
       <a class='current' aria-current='page' href='https://rht-labs.github.io/practice-library/about/'>About</a>
     </li>
   
+    <li>
+          <form id="searchForm">
+                  <input id="searchTerm"> <button onclick="log(event, index)">SEARCH</button>
```

Comment:
I'd recommend putting your click handlers in JavaScript files, not in the HTML. That way, all your JS logic can be centrally located. See [What is Unobtrusive JavaScript?](http://blog.teamtreehouse.com/unobtrusive-javascript-important) 

React and Vue break this rule, but they still use JS-based click handlers behind the scenes - ask me about this sometime if you feel like nerding out over coffee sometime! :smile: